.github/workflows/changelog-check.yml

```
@@ -2,8 +2,9 @@ name: Changelog in sync for packages
 
 on:
   push:
-    branches:
-      - master
+    branches-ignore:
+      - "l10n_master"
+      - "dependabot/*"
```

```suggestion
      - "dependabot/**"
```

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet